shapedetector.py

```diff
@@ -77,15 +80,24 @@ def detect_shapes():
             c = c.astype(int)
             cv2.drawContours(image, [c], -1, (255, 0, 0), 2)
             cv2.putText(image, shape, (cx, cy), cv2.FONT_HERSHEY_SIMPLEX, 0.5, (0, 0, 0), 2)
-
-            # cv2.imshow("Image", image)
-            # cv2.waitKey(0)
-
+	
+    #Draw shapes
```

Comment:
draw what shapes, also put a space between the # in a comment and the text
```suggestion
    # Draw shapes
```